rpcd/etc/openstack_deploy/user_apt_packages.yml

```diff
@@ -0,0 +1,11 @@
+# This file will contain any apt packages that the deployer would like to pin to a particular version.
```

Comment:
It was a file I created to call out the functionality specifically.  You're right that the file being split out for one variable is probably a bit much. I dislike the idea of shoving everything into user_variables, especially when running from 'outside' osad, but in this case I'm not concerned enough to fight for it to stay separated.

And no, I did not encounter errors with this, though I don't remember if I ran with this file present.